package ScienceConfig;

message ScienceItem
{
	required uint32 id 				= 1;	//
	required uint32 pass 			= 2;	//关卡
	required uint32 type 			= 3;	//科技类型
	required uint32 val 			= 4;	//加成值
	required uint32 cd 				= 5;	//解锁cd
	required uint32 cost_coins 		= 6;	//消耗银币
	required uint32 cost_food 		= 7;	//消耗粮食
	required uint32 cost_wood 		= 8;	//消耗木材
	required uint32 cost_iron 		= 9;	//消耗镔铁
	required uint32 capital_cnt 	= 10;	//注资次数
	required uint32 pre_lock_id 	= 11;	//前置解锁,没有为0
	required uint32 speed_cash		= 12;  	//解锁加速所需要的元宝
}

message Science
{
	repeated ScienceItem sciences	= 1;
}